getting_started.md

```diff
@@ -373,32 +393,132 @@ There's a clear drop in diversity in the region of the selective sweep. And as e
 the statistic based on branch-lengths gives a less noisy signal.
 
 
-## Exporting data
+(sec_tskit_getting_started_exporting_data)=
+
+## Saving and exporting data
+
+Tree sequences can be efficiently saved to file using {meth}`TreeSequence.dump`, and
+loaded back again using {func}`tskit.load`. By convention, we use the suffix ``.trees``
+for such files:
+
+```{code-cell} ipython3
+import tskit
+
+ts.dump("data/my_tree_sequence.trees")
+new_ts = tskit.load("data/my_tree_sequence.trees")
+```
+
+It's also possible to export tree sequences to different formats. Note, however, that
+not only are these usually much larger files, but that analysis is usually much faster
+when performed by built-in tskit functions than by exporting and using alternative
+software. If you have a large tree sequence, you should *try to avoid exporting
+to other formats*.
+
+### Newick and Nexus format
+
+The most common format for interchanging tree data is Newick. 
+We can export to a newick format string quite easily. This can be useful for
+interoperating with existing tree processing libraries but is very inefficient for
+large trees. There is also no support for including sites and mutations in the trees.
+
+```{code-cell} ipython3
+small_ts = reduced_ts.keep_intervals([[0, 10000]])
+tree = small_ts.first()
+print(tree.newick(precision=3))
+```
+
+For an entire set of trees, you can use the Nexus file format, which acts as a container
+for a list of Newick format trees, one per line:
+
+```{code-cell} ipython3
+small_ts = small_ts.trim()  # Must trim off the blank region at the end of cut-down ts
+print(small_ts.to_nexus(precision=3))
+```
+
+### VCF
+
+The standard way of interchanging genetic variation data is the Variant Call Format, 
+for which tskit has basic support:
 
-:::{todo}
-Saving in VCF or ms format.
+```{code-cell} ipython3
+import sys
+small_ts.write_vcf(sys.stdout)
+```
+
+The write_vcf method takes a file object as a parameter; to get it to write out to the
+notebook here we ask it to write to stdout.
+
+:::{warning}
+The VCF output in tskit is somewhat primitive and has some gotchas. In particular,
```

Comment:
This isn't worth mentioning now I think, since msprime floating point sites are going to be pretty rare it'll just confuse people.

Reply:
True - I just copied pasted. Will remove.

Reply:
Removed